src/config/models.rs

```diff
@@ -127,6 +127,10 @@ pub struct ConfigOptsServe {
     #[clap(long = "no-autoreload")]
     #[serde(default)]
     pub no_autoreload: bool,
+    #[clap(parse(from_os_str))]
+    pub crt: Option<PathBuf>,
+    #[clap(parse(from_os_str))]
```

Comment:
Let's add these to an arg group to ensure that it is clear to users that both must be declared together.